c10/core/impl/LocalDispatchKeySet.cpp

```diff
@@ -5,7 +5,12 @@
 namespace c10 {
 namespace impl {
 
-// NB: POD, zero initialized!
+// NB: POD, must be zero initialized!
+// Note [TLS Initialization]
+// We want raw_local_dispatch_key_set to be initialized with InplaceOrView key
+// in the included set. But certain Windows compiler (e.g the one used in ARVR tests)
+// only allow TLS to be zero-initialized. So we're working around the problem by adding
+// XOR raw_local_dispatch_key_set.included() with InplaceOrView in DispatchKeyExtractor.h.
```

Comment:
Here's where I think we can describe the scheme in more general terms, rather than strictly a workaround for `InplaceOrView`.